src/bci_build/package/__init__.py

```diff
@@ -118,6 +118,51 @@ def __str__(self) -> str:
         return self.value
 
 
+@enum.unique
+class Registry(enum.Enum):
+    """Supported registries for container image distribution"""
+
+    SUSE_COM = "registry.suse.com"
+    APP_COLL = "dp.apps.rancher.io"
+    OPENSUSE_ORG = "registry.opensuse.org"
```

Comment:
It would look better if it were:

`Registry.SUSE`, `Registry.RANCHER` (or `Registry.RANCHER_PRIME`) and `Registry.OPENSUSE`.

APP_COLL is truncated, and not understandable.
AFAIK, SUSE and openSUSE have only one public registry.
Rancher might have more, but I doubt Rancher Prime has any other.